src/Categories/Bicategory/Adjunction.agda

```diff
@@ -0,0 +1,56 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Bicategory.Adjunction where
+
+open import Categories.Bicategory
+import Categories.Bicategory.Extras as Extras
+open import Categories.Category
+open import Level
+
+private
+  variable
+    o ℓ e t : Level
+
+module _ (𝒞 : Bicategory o ℓ e t) where
+  open Bicategory 𝒞
+
+  record Adjunction (A B : Obj) : Set (o ⊔ ℓ ⊔ e ⊔ t) where
+    private
+      module C = Extras 𝒞
```

Comment:
We should probably pull this module out of the record, and into the enclosing module.

Reply:
Yes. And may as well open it extracting the unitors and ≈.